tests/sasl_plain_ut/sasl_plain_ut.c

```diff
@@ -577,17 +593,16 @@ TEST_FUNCTION(saslplain_challenge_with_NULL_response_bytes_fails)
 TEST_FUNCTION(saslplain_get_interface_returns_the_sasl_plain_mechanism_interface)
 {
 	// arrange
-	amqp_frame_codec_mocks mocks;
 
 	// act
 	const SASL_MECHANISM_INTERFACE_DESCRIPTION* result = saslplain_get_interface();
 
 	// assert
-	ASSERT_ARE_EQUAL(void_ptr, (void_ptr)saslplain_create, (void_ptr)result->concrete_sasl_mechanism_create);
-	ASSERT_ARE_EQUAL(void_ptr, (void_ptr)saslplain_destroy, (void_ptr)result->concrete_sasl_mechanism_destroy);
-	ASSERT_ARE_EQUAL(void_ptr, (void_ptr)saslplain_get_init_bytes, (void_ptr)result->concrete_sasl_mechanism_get_init_bytes);
-	ASSERT_ARE_EQUAL(void_ptr, (void_ptr)saslplain_get_mechanism_name, (void_ptr)result->concrete_sasl_mechanism_get_mechanism_name);
-	ASSERT_ARE_EQUAL(void_ptr, (void_ptr)saslplain_challenge, (void_ptr)result->concrete_sasl_mechanism_challenge);
+	ASSERT_IS_NOT_NULL(result->concrete_sasl_mechanism_create);
```

Comment:
There is a loss of granularity here. 
Before the contents were checked for expected value, now it's only checking for not NULL.

Would suggesting changing it to check the actual expected pointer values, like before.